README.md

```diff
@@ -33,6 +42,30 @@ ETH-duties logs upcoming validator duties to the console in order to find the be
 | nimbus | :heavy_check_mark: | :x: |
 | lodestar | :heavy_check_mark: | :x: |
 
+## Configuration
+
+Most of the available flags are self explanatory. However, some may not be that obvious. Those flags are described in detail in the following chapter.
+
+For all available cli flags please call `eth-duties --help` (see usage examples for further details).
+
+### --mode
+
+The default running mode of `eth-duties` is logging duties to the console (specied with value `log`). However, professional node operators might leverage the tool in their cicd pipelines to e.g. prevent an unintentional client update when your validator is right before proposing a block or part of the sync committee. This is where the different `cicd` modes come into play. You can make your deployment job dependent from the `eth-duties` job so that the deployment job will only run when `eth-duties` exits gracefully with `exit code 0`.  
+
+**Note** If you do not omit attestation duties with `--omit-attestation-duties` these are also considered as valid duties for the cicd modes.
+
+#### cicd-exit
+
+This mode results in a one time check whether one of your supplied validators has an upcoming duty. If there is an upcoming duty the tool exits with `exit code 1`. If there is none the tool exits with `exit code 0`.
+
+#### cicd-wait
+
+This mode results in an ongoing process (similar to the standard behavior) where `eth-duties` checks for upcoming duties until there is no upcoming duty. If there will be no upcoming duty the application exits with `exit code 0`. Compared to the standard logging behavior this process only runs for a certain amount of time (specified with flag `--mode-cicd-waiting-time` (default: 780 seconds, approx. 2 epochs)). If this timeframe ends, `eth-duties` exits with `exit code 1`.
```

Comment:
checks for upcoming duties until there is no upcoming duty -> checks for upcoming duties until there is none